components/grid/grid.html

```diff
@@ -0,0 +1,146 @@
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/typography/typography.html">Typography</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/toast/toast.html">Toast/Snackbar</a>
+                </li>
+                <li class="sidebar-item sidebar-item-active">
+                    <a class="component-link" href="/components/grid/grid.html">Grid</a>
+                </li>
+            </ul>
+            </aside>
+            <main class="main">
+                <div class="component_wrapper">
+                    <h2 class="component_heading">Grid</h2>
+                    <p>
+                        Grid is one of the most efficient CSS utilities to divide block of elements equally in 2D form. We can divide elements in how many parts as we want, but basic partition is <span class="component_description">2 items in grid</span> and <span class="component_description">3 items in grid</span>
+                    </p>
+                    <h3 class="component_heading">2 items in Grid</h3>
+                
+                    <div class="component_wrapper_box">
+                        <div class="grid grid-2">
+                            <div class="grid border-black">
+                                <img src="/assets/avatar_sample.jpg"/>
+                                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                                    Ducimus amet, adipisci sit placeat ipsam fugit quis ut ad illo
+                                    omnis at libero reiciendis deleniti. Autem sequi asperiores culpa aperiam et.</p>
+                            </div>
+                            <div class="grid border-black">
+                                <img src="/assets/avatar_sample.jpg"/>
+                                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                                    Ducimus amet, adipisci sit placeat ipsam fugit quis ut ad illo
+                                    omnis at libero reiciendis deleniti. Autem sequi asperiores culpa aperiam et.</p>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="component_iframe">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%281%2C22%2C39%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=22px&dsblur=68px&wc=true&wa=true&pv=36px&ph=47px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid%2520grid-2%2522%253E%250A%2509%253Cdiv%2520class%253D%2522component%2522%253Econtent...%253C%252Fdiv%253E%250A%2509%253Cdiv%2520class%253D%2522component%2522%253Econtent...%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
```

Comment:
Because you are using carbon.sh you can maybe add loading="lazy" and also if you want to make it responsive you can add width:100%.Just a suggestion.